src/transport/SessionManager.cpp

```diff
@@ -337,15 +337,15 @@ CHIP_ERROR SessionManager::SendPreparedMessage(const SessionHandle & sessionHand
 
 void SessionManager::ExpirePairing(const SessionHandle & sessionHandle)
```

Comment:
The existing expectation that upon completion of this call, that the SecureSession object will be reclaimed and a slot made available in the session table is now no longer true nor guaranteed. That needs to be well documented (this function seems to have no description in the header??).